### ISSUE TITLE
ImagePicker widget does not work in dynamically added inlines

### ISSUE THREAD

**Issue by o-zander** [divio/django-filer#743]
I have a parent model and an image model with a foreign key to the parent and an image field. The parent's admin includes the image model as an inline. I got nearly crazy because I can't use the image picker. If I clicked add image (to get a new inline) and tried to drop an image on the field but the image was shown in the browser instead of being uploaded. So I tried to add an image over the folder structure. But after choosing one, it didn't get inserted in the field. So I thought it has something to do with mit setup. After hours of trying it out with different admin configurations and test models I raised the extra attribute of my image inline admin from zero to one. And surprisingly I could drop images! But if add an inline dynamically with the add button the functionality is gone.

**Long story short: a file/image field in an inline can't be used on dynamically added inlines.**

**Comment by stefanfoulis** [comment]
@o-zander thanks for reporting. We'll try to get a fix out for this regression asap.

**Comment by o-zander** [comment]
Great :) I would help but frontend related issues are not my strength ;)

**Comment by stefanfoulis** [comment]
probably fixed in #670
